docs/source/quickstart.mdx

```diff
@@ -34,15 +35,15 @@ ppo_trainer = PPOTrainer(config, model, model_ref, tokenizer)
 
 # 3. encode a query
 query_txt = "This morning I went to the "
-query_tensor = tokenizer.encode(query_txt, return_tensors="pt")
+query_tensor = tokenizer.encode(query_txt, return_tensors="pt").to(model.pretrained_model.device)
 
 # 4. generate model response
 response_tensor  = respond_to_batch(model, query_tensor)
 response_txt = tokenizer.decode(response_tensor[0,:])
 
 # 5. define a reward for response
 # (this could be any reward such as human feedback or output from another model)
-reward = [torch.tensor(1.0)]
+reward = [torch.tensor(1.0, device=model.pretrained_model.device)]
```

Comment:
Without the `, device=model.pretrained_model.device` the code still works but the reward must be transferred to the device somewhere in the trainer.